shader/lighting/tonemapping.frag

```diff
@@ -10,6 +10,31 @@
 
 #include "upscale/lanczos.glsl"
 
+#if defined(COMPUTE)
+layout(local_size_x = 8, local_size_y = 8, local_size_z = 1) in;
+layout(binding = 2) uniform writeonly image2D tonemappedOutput;
+layout(r32f, binding = 3) uniform writeonly image2D hdrLumaOutput;
+
+/* layout:
+0 1 4 5 ... 
+2 3 6 7 ...
+16 17 ...
+18 19 ...
+*/
+uvec2 getRemmappedPixelPos(uvec2 groupID, uint localThreadIndex)
```

Comment:
Does this remapping works? So far wasn't able to find any performance gain on my machine (Intel + NVidia)

Reply:
https://developer.nvidia.com/blog/optimizing-compute-shaders-for-l2-locality-using-thread-group-id-swizzling/
It should increase perfornamce on gpus that have vram throughput as a bottleneck

Reply:
I'm aware of that article. Problem, that so far wasn't able to confirm it: even with locked GPU-frequency performance change in memory-bound algorithm (fog occlusion) is in a noise range.

And in general doesn't "feel" right, to reason about cache-locality of non-opaque data with unknown(to application) memory layout.